src/hotspot/share/runtime/globals.hpp

```diff
@@ -2027,6 +2027,16 @@ const intx ObjectAlignmentInBytes = 8;
              "Use the FP register for holding the frame pointer "           \
              "and not as a general purpose register.")                      \
                                                                             \
+  product(intx, LogAsyncInterval, 300,                                      \
+          "Milliseconds between asynchronous log flushing")                 \
+                                                                            \
+  product(bool, AsyncLogging, false,                                        \
+          "Enble asynchronous GC logging")                                  \
+                                                                            \
+  product(size_t, GCLogBufferSize, 2*K,                                     \
```

Comment:
This PR is for UL, not only GC log. So it should be renamed.

Reply:
ack. I will rename it AsyncLogBufferSize.